UCAB - IllegalStateException No match found for dlager get command

**Beschreibe den Bug oder Fehler**

**Wie kann der Bug oder Fehler reproduziert werden?**

**Erwartetes Verhalten**

**Screenshots**
```
[14:50:51] [Timer-13/INFO] [com.rettichlp.unicacityaddon.UnicacityAddon]: UPlayer sent chat message: /mobile
[14:51:09] [Timer-12/INFO] [com.rettichlp.unicacityaddon.UnicacityAddon]: APIResponse - 200 [http://rettichlp.de:8888/unicacityaddon/v1/TOKEN/broadcast/queue]
[14:51:11] [Client thread/INFO] [com.rettichlp.unicacityaddon.UnicacityAddon]: UPlayer sent chat message: /hkasse
[14:51:11] [Client thread/INFO] [net.labymod.core_implementation.mc112.gui.GuiChatAdapter]: [CHAT] ? 14:51:11 » Drogenlager vom Haus 461:
[14:51:11] [Client thread/INFO] [net.labymod.core_implementation.mc112.gui.GuiChatAdapter]: [CHAT] ? 14:51:11 »  - Kokain: 40g
[14:51:11] [Client thread/INFO] [net.labymod.core_implementation.mc112.gui.GuiChatAdapter]: [CHAT] ? 14:51:11 »  - Marihuana: 0g
[14:51:11] [Client thread/INFO] [net.labymod.core_implementation.mc112.gui.GuiChatAdapter]: [CHAT] ? 14:51:11 »  - Methamphetamin: 0g
[14:51:11] [Client thread/INFO] [net.labymod.core_implementation.mc112.gui.GuiChatAdapter]: [CHAT] ? 14:51:11 »  - Hustensaft: 0g
[14:51:11] [Client thread/INFO] [net.labymod.core_implementation.mc112.gui.GuiChatAdapter]: [CHAT] ? 14:51:11 »  - Schmerzmittel: 50g
[14:51:11] [Client thread/INFO] [net.labymod.core_implementation.mc112.gui.GuiChatAdapter]: [CHAT] ? 14:51:11 »  - Antibiotika: 0g
[14:51:18] [Client thread/INFO] [com.rettichlp.unicacityaddon.UnicacityAddon]: UPlayer sent chat message: /hkasse
[14:51:18] [Client thread/INFO] [com.rettichlp.unicacityaddon.UnicacityAddon]: HOUSE_STORAGE_SUCCESSFUL_INTERACTION found with waiting command: /drogenlager get schmerzmittel 50 0
[14:51:18] [Client thread/ERROR] [FML]: Exception caught during firing event net.minecraftforge.client.event.ClientChatReceivedEvent@310d9735:
java.lang.IllegalStateException: No match found
	at java.util.regex.Matcher.getMatchedGroupIndex(Unknown Source) ~[?:1.8.0_311]
	at java.util.regex.Matcher.group(Unknown Source) ~[?:1.8.0_311]
	at com.rettichlp.unicacityaddon.events.house.HouseDataEventHandler.onClientChatReceived(HouseDataEventHandler.java:82) ~[HouseDataEventHandler.class:1.8.0]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_23_HouseDataEventHandler_onClientChatReceived_ClientChatReceivedEvent.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182) [EventBus.class:?]
	at net.minecraftforge.event.ForgeEventFactory.onClientChat(ForgeEventFactory.java:429) [ForgeEventFactory.class:?]
	at net.minecraft.client.network.NetHandlerPlayClient.func_147251_a(NetHandlerPlayClient.java:843) [brz.class:?]
	at net.minecraft.network.play.server.SPacketChat.func_148833_a(SourceFile:41) [in.class:?]
	at net.minecraft.network.play.server.SPacketChat.func_148833_a(SourceFile:10) [in.class:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:22) [hv$1.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_311]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_311]
	at net.minecraft.util.Util.func_181617_a(Util.java:47) [h.class:?]
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1088) [bib.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:398) [bib.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:123) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_311]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_311]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_311]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_311]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_311]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_311]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_311]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_311]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
[14:51:18] [Client thread/ERROR] [FML]: Index: 13 Listeners:
[14:51:18] [Client thread/ERROR] [FML]: 0: HIGHEST
[14:51:18] [Client thread/ERROR] [FML]: 1: ASM: com.rettichlp.unicacityaddon.events.faction.badfaction.blacklist.BlacklistEventHandler@452bcee onBlacklistShown(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 2: ASM: com.rettichlp.unicacityaddon.events.faction.polizei.WantedEventHandler@7b601e55 onWantedsDeleted(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 3: ASM: com.rettichlp.unicacityaddon.events.faction.polizei.WantedEventHandler@7b601e55 onWantedsShown(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 4: ASM: com.rettichlp.unicacityaddon.events.faction.polizei.WantedEventHandler@7b601e55 onWantedsGiven(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 5: ASM: com.rettichlp.unicacityaddon.events.faction.ContractEventHandler@5a33ba9b onContractRemoved(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 6: ASM: com.rettichlp.unicacityaddon.events.faction.ContractEventHandler@5a33ba9b onContractSet(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 7: ASM: com.rettichlp.unicacityaddon.events.faction.ContractEventHandler@5a33ba9b onHitlistShown(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 8: NORMAL
[14:51:18] [Client thread/ERROR] [FML]: 9: ASM: com.rettichlp.unicacityaddon.events.faction.ReinforcementEventHandler@60d46bd7 onClientChatReceive(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 10: ASM: com.rettichlp.unicacityaddon.events.job.JobEventHandler@2248583e onChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 11: ASM: com.rettichlp.unicacityaddon.events.ABuyEventHandler@451f5e53 onChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 12: ASM: com.rettichlp.unicacityaddon.events.faction.polizei.HQMessageEventHandler@3463851a onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 13: ASM: com.rettichlp.unicacityaddon.events.house.HouseDataEventHandler@7f8b9fa7 onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 14: ASM: com.rettichlp.unicacityaddon.events.faction.rettungsdienst.FirstAidEventHandler@526ee5e6 onFirstAidReceipt(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 15: ASM: com.rettichlp.unicacityaddon.events.faction.badfaction.PlantEventHandler@3f51fcf5 onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 16: ASM: com.rettichlp.unicacityaddon.events.job.FishermanEventHandler@2aa28a01 onClientChatReceive(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 17: ASM: com.rettichlp.unicacityaddon.events.faction.badfaction.GiftEigenbedarfEventHandler@8d1b425 onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 18: ASM: com.rettichlp.unicacityaddon.events.faction.EquipEventHandler@7d7a13a7 onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 19: ASM: com.rettichlp.unicacityaddon.events.faction.rettungsdienst.ServiceMessageEventHandler@3a778c6f onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 20: ASM: com.rettichlp.unicacityaddon.events.FriendJoinEventHandler@53e36530 onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 21: ASM: com.rettichlp.unicacityaddon.events.WeaponClickEventHandler@29c947d0 onChat(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 22: ASM: com.rettichlp.unicacityaddon.events.faction.FactionInfoEventHandler@7f277cf6 onChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 23: ASM: com.rettichlp.unicacityaddon.events.MobileEventHandler@68172eeb onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 24: ASM: com.rettichlp.unicacityaddon.events.house.HouseInteractionEventHandler@3aac686f onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 25: ASM: com.rettichlp.unicacityaddon.events.faction.AEquipEventHandler@b95e63a onChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 26: ASM: com.rettichlp.unicacityaddon.events.faction.badfaction.DBankMessageEventHandler@5592464a onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 27: ASM: com.rettichlp.unicacityaddon.events.faction.rettungsdienst.FireEventHandler@3bd42eef onReviveStart(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 28: ASM: com.rettichlp.unicacityaddon.events.CarEventHandler@223973a9 onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 29: ASM: com.rettichlp.unicacityaddon.events.team.ReportEventHandler@6b907eec onClientChatReceive(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 30: ASM: com.rettichlp.unicacityaddon.events.faction.ShareLocationEventHandler@7d4a95d4 onClientChatReceive(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 31: ASM: com.rettichlp.unicacityaddon.events.MoneyEventHandler@4d760ddb onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 32: ASM: com.rettichlp.unicacityaddon.events.TimerEventHandler@5ffac740 onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 33: ASM: com.rettichlp.unicacityaddon.events.faction.rettungsdienst.ReviveEventHandler@5d6850f7 onReviveStart(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 34: ASM: com.rettichlp.unicacityaddon.events.HotkeyEventHandler@607fa188 onClientChatReceive(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 35: ASM: com.rettichlp.unicacityaddon.events.faction.badfaction.DrugInteractionEventHandler@12464b1c onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 36: ASM: com.rettichlp.unicacityaddon.events.faction.EmergencyServiceEventHandler@1f1d7c52 onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 37: ASM: com.rettichlp.unicacityaddon.events.house.HouseRenterEventHandler@64e5f67b onChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 38: ASM: com.rettichlp.unicacityaddon.events.AccountEventHandler@5ddc735b onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 39: ASM: com.rettichlp.unicacityaddon.events.KarmaMessageEventHandler@4861df70 onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 40: ASM: com.rettichlp.unicacityaddon.events.faction.terroristen.BombTimerEventHandler@2122eb64 onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 41: ASM: com.rettichlp.unicacityaddon.events.NavigationEventHandler@142c0c3e onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 42: ASM: com.rettichlp.unicacityaddon.events.faction.badfaction.blacklist.BlacklistModifyEventHandler@11a0aee8 onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 43: ASM: com.rettichlp.unicacityaddon.events.faction.badfaction.BannerEventHandler@12cd1fe6 onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 44: ASM: com.rettichlp.unicacityaddon.events.faction.rettungsdienst.MedicationEventHandler@1d88414a onRecipeAcceptFeedback(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 45: ASM: com.rettichlp.unicacityaddon.events.faction.rettungsdienst.MedicationEventHandler@1d88414a onRecipeGiveFeedback(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 46: ASM: com.rettichlp.unicacityaddon.events.faction.AFbankEinzahlenEventHandler@4968bd4e onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/ERROR] [FML]: 47: ASM: com.rettichlp.unicacityaddon.events.DeathsKillsEventHandler@745955ba onClientChatReceived(Lnet/minecraftforge/client/event/ClientChatReceivedEvent;)V
[14:51:18] [Client thread/FATAL] [net.minecraft.client.Minecraft]: Error executing task
java.util.concurrent.ExecutionException: java.lang.IllegalStateException: No match found
	at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_311]
	at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_311]
	at net.minecraft.util.Util.func_181617_a(Util.java:48) [h.class:?]
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1088) [bib.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:398) [bib.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:123) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_311]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_311]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_311]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_311]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_311]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_311]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_311]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_311]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
Caused by: java.lang.IllegalStateException: No match found
	at java.util.regex.Matcher.getMatchedGroupIndex(Unknown Source) ~[?:1.8.0_311]
	at java.util.regex.Matcher.group(Unknown Source) ~[?:1.8.0_311]
	at com.rettichlp.unicacityaddon.events.house.HouseDataEventHandler.onClientChatReceived(HouseDataEventHandler.java:82) ~[HouseDataEventHandler.class:1.8.0]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_23_HouseDataEventHandler_onClientChatReceived_ClientChatReceivedEvent.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182) ~[EventBus.class:?]
	at net.minecraftforge.event.ForgeEventFactory.onClientChat(ForgeEventFactory.java:429) ~[ForgeEventFactory.class:?]
	at net.minecraft.client.network.NetHandlerPlayClient.func_147251_a(NetHandlerPlayClient.java:843) ~[brz.class:?]
	at net.minecraft.network.play.server.SPacketChat.func_148833_a(SourceFile:41) ~[in.class:?]
	at net.minecraft.network.play.server.SPacketChat.func_148833_a(SourceFile:10) ~[in.class:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:22) ~[hv$1.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[?:1.8.0_311]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:1.8.0_311]
	at net.minecraft.util.Util.func_181617_a(Util.java:47) ~[h.class:?]
	... 17 more
[14:51:18] [pool-7-thread-1/ERROR] [net.labymod.support.util.CapturePrintStream]: Catching
java.lang.NumberFormatException: For input string: "rainbow"
	at java.lang.NumberFormatException.forInputString(Unknown Source) ~[?:1.8.0_311]
	at java.lang.Integer.parseInt(Unknown Source) ~[?:1.8.0_311]
	at java.lang.Integer.parseInt(Unknown Source) ~[?:1.8.0_311]
	at net.labymod.user.cosmetic.geometry.effect.effects.GeometryColor.parse(GeometryColor.java:20) ~[GeometryColor.class:?]
	at net.labymod.user.cosmetic.geometry.effect.GeometryEffect.load(GeometryEffect.java:20) ~[GeometryEffect.class:?]
	at net.labymod.user.cosmetic.geometry.effect.EnumEffectType.create(EnumEffectType.java:46) ~[EnumEffectType.class:?]
	at net.labymod.user.cosmetic.geometry.effect.EnumEffectType.createEffect(EnumEffectType.java:42) ~[EnumEffectType.class:?]
	at net.labymod.user.cosmetic.geometry.BlockBenchLoader.addMapping(BlockBenchLoader.java:207) [BlockBenchLoader.class:?]
	at net.labymod.user.cosmetic.geometry.BlockBenchLoader.addModel(BlockBenchLoader.java:162) [BlockBenchLoader.class:?]
	at net.labymod.user.cosmetic.geometry.BlockBenchLoader.addGroup(BlockBenchLoader.java:86) [BlockBenchLoader.class:?]
	at net.labymod.user.cosmetic.geometry.BlockBenchLoader.addGroup(BlockBenchLoader.java:89) [BlockBenchLoader.class:?]
	at net.labymod.user.cosmetic.geometry.BlockBenchLoader.addGroup(BlockBenchLoader.java:89) [BlockBenchLoader.class:?]
	at net.labymod.user.cosmetic.geometry.BlockBenchLoader.addGroup(BlockBenchLoader.java:89) [BlockBenchLoader.class:?]
	at net.labymod.user.cosmetic.geometry.BlockBenchLoader.<init>(BlockBenchLoader.java:57) [BlockBenchLoader.class:?]
	at net.labymod.user.cosmetic.geometry.GeometryLoader.toBlockBenchLoader(GeometryLoader.java:59) [GeometryLoader.class:?]
	at net.labymod.user.cosmetic.geometry.GeometryCosmetic.addModels(GeometryCosmetic.java:49) [GeometryCosmetic.class:?]
	at net.labymod.user.cosmetic.ModelCosmetics$1.load(ModelCosmetics.java:95) [ModelCosmetics$1.class:?]
	at net.labymod.user.cosmetic.remote.RemoteCosmeticLoader.loadRemote(RemoteCosmeticLoader.java:181) [RemoteCosmeticLoader.class:?]
	at net.labymod.user.cosmetic.remote.RemoteCosmeticLoader.discover(RemoteCosmeticLoader.java:106) [RemoteCosmeticLoader.class:?]
	at net.labymod.user.UserManager.handleJsonString(UserManager.java:477) [UserManager.class:?]
	at net.labymod.user.UserManager.access$500(UserManager.java:63) [UserManager.class:?]
	at net.labymod.user.UserManager$5.run(UserManager.java:388) [UserManager$5.class:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_311]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_311]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_311]
[14:51:18] [pool-7-thread-1/ERROR] [net.labymod.support.util.CapturePrintStream]: Catching
java.lang.NumberFormatException: For input string: "rainbow"
	at java.lang.NumberFormatException.forInputString(Unknown Source) ~[?:1.8.0_311]
	at java.lang.Integer.parseInt(Unknown Source) ~[?:1.8.0_311]
	at java.lang.Integer.parseInt(Unknown Source) ~[?:1.8.0_311]
	at net.labymod.user.cosmetic.geometry.effect.effects.GeometryColor.parse(GeometryColor.java:20) ~[GeometryColor.class:?]
	at net.labymod.user.cosmetic.geometry.effect.GeometryEffect.load(GeometryEffect.java:20) ~[GeometryEffect.class:?]
	at net.labymod.user.cosmetic.geometry.effect.EnumEffectType.create(EnumEffectType.java:46) ~[EnumEffectType.class:?]
	at net.labymod.user.cosmetic.geometry.effect.EnumEffectType.createEffect(EnumEffectType.java:42) ~[EnumEffectType.class:?]
	at net.labymod.user.cosmetic.geometry.BlockBenchLoader.addMapping(BlockBenchLoader.java:207) [BlockBenchLoader.class:?]
	at net.labymod.user.cosmetic.geometry.BlockBenchLoader.addModel(BlockBenchLoader.java:162) [BlockBenchLoader.class:?]
	at net.labymod.user.cosmetic.geometry.BlockBenchLoader.addGroup(BlockBenchLoader.java:86) [BlockBenchLoader.class:?]
	at net.labymod.user.cosmetic.geometry.BlockBenchLoader.addGroup(BlockBenchLoader.java:89) [BlockBenchLoader.class:?]
	at net.labymod.user.cosmetic.geometry.BlockBenchLoader.addGroup(BlockBenchLoader.java:89) [BlockBenchLoader.class:?]
	at net.labymod.user.cosmetic.geometry.BlockBenchLoader.addGroup(BlockBenchLoader.java:89) [BlockBenchLoader.class:?]
	at net.labymod.user.cosmetic.geometry.BlockBenchLoader.<init>(BlockBenchLoader.java:57) [BlockBenchLoader.class:?]
	at net.labymod.user.cosmetic.geometry.GeometryLoader.toBlockBenchLoader(GeometryLoader.java:59) [GeometryLoader.class:?]
	at net.labymod.user.cosmetic.geometry.GeometryCosmetic.addModels(GeometryCosmetic.java:49) [GeometryCosmetic.class:?]
	at net.labymod.user.cosmetic.ModelCosmetics$1.load(ModelCosmetics.java:95) [ModelCosmetics$1.class:?]
	at net.labymod.user.cosmetic.remote.RemoteCosmeticLoader.loadRemote(RemoteCosmeticLoader.java:181) [RemoteCosmeticLoader.class:?]
	at net.labymod.user.cosmetic.remote.RemoteCosmeticLoader.discover(RemoteCosmeticLoader.java:106) [RemoteCosmeticLoader.class:?]
	at net.labymod.user.UserManager.handleJsonString(UserManager.java:477) [UserManager.class:?]
	at net.labymod.user.UserManager.access$500(UserManager.java:63) [UserManager.class:?]
	at net.labymod.user.UserManager$5.run(UserManager.java:388) [UserManager$5.class:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_311]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_311]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_311]
[14:51:23] [Client thread/INFO] [com.rettichlp.unicacityaddon.UnicacityAddon]: UPlayer sent chat message: /hkasse
[14:51:24] [Client thread/INFO] [net.labymod.core_implementation.mc112.gui.GuiChatAdapter]: [CHAT] ? 14:51:24 » Fehler: Du hast nicht genug Drogen/Medizin dabei.
[14:51:26] [Client thread/INFO] [com.rettichlp.unicacityaddon.UnicacityAddon]: UPlayer sent chat message: /hkasse
[14:51:26] [Client thread/INFO] [com.rettichlp.unicacityaddon.UnicacityAddon]: HOUSE_STORAGE_SUCCESSFUL_INTERACTION found with waiting command: /drogenlager drop schmerzmittel 50 0
[14:51:27] [Client thread/INFO] [net.labymod.core_implementation.mc112.gui.GuiChatAdapter]: [CHAT] ? 14:51:27 » [Haus] Du hast 50g Schmerzmittel ins Drogenlager gelegt.
[14:51:27] [Timer-15/INFO] [com.rettichlp.unicacityaddon.UnicacityAddon]: UPlayer sent chat message: /afk
[14:51:27] [Client thread/INFO] [net.labymod.core_implementation.mc112.gui.GuiChatAdapter]: [CHAT] ? 14:51:27 » §r§b  Info: Das Addon versucht dich anschließend in den AFK Modus zu setzen.
[14:51:27] [Client thread/INFO] [net.labymod.core_implementation.mc112.gui.GuiChatAdapter]: [CHAT] ? 14:51:27 » Du kannst noch nicht in den AFK Modus wechseln.
[14:51:27] [Client thread/INFO] [net.labymod.core_implementation.mc112.gui.GuiChatAdapter]: [CHAT] ? 14:51:27 » Bitte warte ein wenig, bis du erneut einen Befehl ausführst.
```

**Weitere Informationen (bitte ausfüllen):**
Minecraft Version: `1.12.2`
Addon Version: `1.8.0`

--- a/src/main/java/com/rettichlp/unicacityaddon/events/house/HouseDataEventHandler.java
+++ b/src/main/java/com/rettichlp/unicacityaddon/events/house/HouseDataEventHandler.java
@@ -79,9 +79,9 @@ public class HouseDataEventHandler {
 
             Matcher drugStorageRemoveCommandMatcher = PatternHandler.HOUSE_STORAGE_REMOVE_COMMAND_PATTERN.matcher(waitingCommand);
             if (drugStorageRemoveCommandMatcher.find()) {
-                int amount = Integer.parseInt(drugStorageAddCommandMatcher.group("amount"));
-                DrugType drugType = DrugType.getDrugType(drugStorageAddCommandMatcher.group("drugType"));
-                DrugPurity drugPurity = DrugPurity.getDrugPurity(drugStorageAddCommandMatcher.group("drugPurity"));
+                int amount = Integer.parseInt(drugStorageRemoveCommandMatcher.group("amount"));
+                DrugType drugType = DrugType.getDrugType(drugStorageRemoveCommandMatcher.group("drugType"));
+                DrugPurity drugPurity = DrugPurity.getDrugPurity(drugStorageRemoveCommandMatcher.group("drugPurity"));
 
                 HouseData houseData = HouseDataManager.getHouseData(lastCheckedHouseNumber).removeFromStorage(drugType, drugPurity, amount);
                 HouseDataManager.updateHouseData(lastCheckedHouseNumber, houseData);

